src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -113,6 +113,7 @@ public void doPost_invalidLink_badRequest() throws Exception {
   public void doPost_urlAlreadyInDatabase_shouldReturnLecture() throws Exception {
```

Comment:
is there a test for when the language parameter is missing?

apologies if it was included in a previous PR

Reply:
There is one in this PR for `TranscriptParser` but not for the `LectureServlet`. I'll add one here too. Thanks!